applications/nrf_desktop/src/modules/failsafe.c

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+#include <hal/nrf_power.h>
+#include <storage/flash_map.h>
+
+#define MODULE failsafe
+#include "module_state_event.h"
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(MODULE);
```

Comment:
We should add the log level here (or do not generate it in Kconfig)

Reply:
done